tests/unit/providers/wikipedia/backends/test_elastic.py

```diff
@@ -1,86 +1,59 @@
-"""Unit tests for the Merino v1 suggest API endpoint for the Wikipedia provider."""
+"""Unit tests for the Merino v1 suggest API endpoint for the Elastic Backend."""
+from typing import Optional
 from unittest.mock import AsyncMock
 
 import pytest
 from elasticsearch import AsyncElasticsearch
 from pytest_mock import MockerFixture
 
-from merino.config import settings
 from merino.exceptions import BackendError
-from merino.providers.wikipedia.backends.elastic import SUGGEST_ID, ElasticBackend
-from merino.providers.wikipedia.backends.fake_backends import FakeEchoWikipediaBackend
-from merino.providers.wikipedia.provider import (
-    ADVERTISER,
-    ICON,
-    Provider,
-    WikipediaSuggestion,
+from merino.providers.wikipedia.backends.elastic import (
+    SUGGEST_ID,
+    ElasticBackend,
+    ElasticBackendError,
 )
-from tests.unit.types import SuggestionRequestFixture
 
-
-@pytest.fixture(name="wikipedia")
-def fixture_wikipedia() -> Provider:
-    """Return a Wikipedia provider that uses a test backend."""
-    return Provider(backend=FakeEchoWikipediaBackend())
+FAKE_ELASTIC_CLOUD_ID = (
+    "cluster:dXMtZWFzdC0xLmF3cy5mb3VuZC5pbyQ0ZmE4ODIxZTc1NjM0MDMyYmVk"
+    "MWNmMjIxMTBlMmY5NyQ0ZmE4ODIxZTc1NjM0MDMyYmVkMWNmMjIxMTBlMmY5Ng=="
+)
 
 
 @pytest.fixture(name="es_backend")
 def fixture_es_backend() -> ElasticBackend:
     """Return an ES backend instance."""
     return ElasticBackend(
-        cloud_id=settings.providers.wikipedia.es_cloud_id,
-        user=settings.providers.wikipedia.es_user,
-        password=settings.providers.wikipedia.es_password,
+        url="http://localhost:9200",
+        cloud_id=None,
     )
 
 
-def test_enabled_by_default(wikipedia: Provider) -> None:
-    """Test for the enabled_by_default method."""
-    assert wikipedia.enabled_by_default
-
+def test_initialize_es_backend_cloud_id_success():
+    """Test that the ElasticBackend initializes successfully with a cloud_id."""
+    es = ElasticBackend(cloud_id=FAKE_ELASTIC_CLOUD_ID)
+    assert es
 
-def test_hidden(wikipedia: Provider) -> None:
-    """Test for the hidden method."""
-    assert not wikipedia.hidden()
 
-
-@pytest.mark.asyncio
-async def test_shutdown(wikipedia: Provider, mocker: MockerFixture) -> None:
-    """Test for the shutdown method."""
-    spy = mocker.spy(FakeEchoWikipediaBackend, "shutdown")
-    await wikipedia.shutdown()
-    spy.assert_called_once()
+def test_initialize_es_backend_url_success():
+    """Test that the ElasticBackend initializes successfully with a URL."""
+    es = ElasticBackend(url="http://localhost:9200")
+    assert es
 
 
-@pytest.mark.asyncio
 @pytest.mark.parametrize(
-    ["query", "expected_title"],
-    [("foo", "foo"), ("foo bar", "foo_bar"), ("foØ bÅr", "fo%C3%98_b%C3%85r")],
+    ["url", "cloud_id"],
+    [
+        (None, None),
+        (
+            "https://localhost:9200",
+            FAKE_ELASTIC_CLOUD_ID,
+        ),
+    ],
```

Comment:
You could add ids if you think it adds clarity?
```suggestion
    ],
    ids=["without_url_and_cloud_id", "with_url_and_cloud_id"]
```

Reply:
After turning `FAKE_ELASTIC_CLOUD_ID` into a fixture, I decided to not parameterize this function, so split this into 2 test functions instead.